packages/client/hmi-client/src/components/widgets/tera-related-documents.vue

```diff
@@ -201,13 +200,7 @@ const sendForExtractions = async () => {
 	// Model extraction
 	if (props.assetType === AssetType.Model && selectedResourceId) {
 		await extractVariables(selectedResourceId, [props.assetId]);
-		const isAligned = await alignModel(props.assetId, selectedResourceId);
-		if (isAligned) {
-			logger.success('Model aligned after variable extraction.');
-			emit('enriched');
-		} else {
-			logger.warn('Model was not aligned after variable extraction. Please try again.');
-		}
+		emit('enriched');
```

Comment:
I dunno if this should even be here since that call is async and just queues the work